incubator-site/_site/community/index.html

```diff
@@ -0,0 +1,206 @@
+    </tr>
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+      <td><strong>Commits</strong>: <a href="&#109;&#097;&#105;&#108;&#116;&#111;:&#099;&#111;&#109;&#109;&#105;&#116;&#115;&#045;&#115;&#117;&#098;&#115;&#099;&#114;&#105;&#098;&#101;&#064;&#109;&#121;&#114;&#105;&#097;&#100;&#046;&#105;&#110;&#099;&#117;&#098;&#097;&#116;&#111;&#114;&#046;&#097;&#112;&#097;&#099;&#104;&#101;&#046;&#111;&#114;&#103;&#063;&#115;&#117;&#098;&#106;&#101;&#099;&#116;&#061;&#083;&#117;&#098;&#115;&#099;&#114;&#105;&#098;&#101;&#038;&#098;&#111;&#100;&#121;&#061;&#083;&#117;&#098;&#115;&#099;&#114;&#105;&#098;&#101;">Subscribe</a> | <a href="&#109;&#097;&#105;&#108;&#116;&#111;:&#099;&#111;&#109;&#109;&#105;&#116;&#115;&#045;&#117;&#110;&#115;&#117;&#098;&#115;&#099;&#114;&#105;&#098;&#101;&#064;&#109;&#121;&#114;&#105;&#097;&#100;&#046;&#105;&#110;&#099;&#117;&#098;&#097;&#116;&#111;&#114;&#046;&#097;&#112;&#097;&#099;&#104;&#101;&#046;&#111;&#114;&#103;&#063;&#115;&#117;&#098;&#106;&#101;&#099;&#116;&#061;&#085;&#110;&#115;&#117;&#098;&#115;&#099;&#114;&#105;&#098;&#101;&#038;&#098;&#111;&#100;&#121;&#061;&#085;&#110;&#115;&#117;&#098;&#115;&#099;&#114;&#105;&#098;&#101;">Unsubscribe</a> | <a href="http://www.mail-archive.com/commits@myriad.incubator.apache.org/">Archives</a></td>
+    </tr>
+  </tbody>
+</table>
+
+<h4 id="work-the-code">Work the Code</h4>
+
+<table>
+  <tbody>
+    <tr>
+      <td><a href="https://cwiki.apache.org/myriad">Read about Myriad:</a> We use Confluence for documentation.</td>
+    </tr>
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+      <td><a href="https://issues.apache.org/jira/browse/MYRIAD">Browse or Submit Bugs:</a> We use JIRA for issue management.</td>
+    </tr>
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+      <td><a href="https://git-wip-us.apache.org/repos/asf?p=incubator-myriad.git">Official Repo</a> | <a href="https://github.com/apache/incubator-myriad">Github mirror:</a> Checkout the code.</td>
+    </tr>
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+      <td><a href="https://github.com/apache/incubator-myriad/pulls">Submit a patch:</a> Use Github for pull requests.</td>
+    </tr>
+  </tbody>
+</table>
+
+<h4 id="hangout-with-developers">Hangout with Developers</h4>
+
+<p>Apache Myriad developers gather bi-weekly for a Google Hangout to sync up commits, talk about features, upcoming events, and more. Subscribe to the dev mailing list to find out when and receive links to join before it starts.</p>
```

Comment:
Can you mention that we meet bi-weekly "on Wednesdays" and link to the Meeting minutes doc: https://docs.google.com/a/mesosphere.io/document/d/1JGmJrgeg98bHw_0_sSRmyX6WiAe13OdErcFlaz6Aa04/edit#heading=h.aaws1n2fsgsi